controller/src/main/java/io/pravega/controller/server/security/auth/handler/impl/AclAuthorizer.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server.security.auth.handler.impl;
+
+import io.pravega.auth.AuthHandler;
+
+/**
+ * Authorizes resources based on supplied ACLs.
+ */
+abstract class AclAuthorizer {
```

Comment:
is it possible that some ACLs are defined in legacy format while others are in new format? or is it expected to be consistent for all ACLs?

Reply:
As per current implementation, all ACLs must be specified in the new format. Any ACLs specified in legacy format won't work.

Reply:
Yes, that is right.